src/devices/sound/lc82310.cpp

```diff
@@ -0,0 +1,296 @@
+// license:BSD-3-Clause
+// copyright-holders:windyfairy
+/**********************************************************************
+
+    Sanyo LC82310 MP3 decoder
+
+**********************************************************************/
+
+#include "lc82310.h"
+#include "mp3_audio.h"
```

Comment:
Remember you need to `#include "emu.h"` at the top so it works properly with prefix headers disabled.